wp-content/themes/amplify/amplify-feed.php

```diff
@@ -161,7 +167,11 @@
             _e( '<p class="error">You don\'t have any recent links or the link roundups plugin is not active.</p>', 'link-roundups' );
         } // end recent links
 
-        ?>
+        if( isset( $_GET['gform_id'] ) ) { ?>
+
+            <a target="_blank" class="submit-idea-btn btn" href="https://docs.google.com/forms/d/e/<?php echo $_GET['gform_id']; ?>/viewform">Submit a story idea</a>
```

Comment:
At the minimum, this section needs to perform escaping on `gform_id` to make sure that it has the form of a Google Form ID. I haven't seen an official definition, but thus far I've seen 0-9, a-z, A-Z, - and _ in form IDs.

But that still doesn't prevent someone from supplying their own form ID here.

Reply:
I guess another alternative would be to create a conditional to swap form ID's for each different project. But... that would require a deploy each time we wanted to add a new form for a new project.

Reply:
We could do that for _this_ project, and then upgrade it when we do the _next_ project.

Reply:
By "upgrade it" I mean put in term meta fields.

Reply:
Either way, I agree, we might as well just add new term meta fields for the LR saved links tags.

Reply:
@benlk https://github.com/INN/umbrella-inndev/pull/138/commits/338642444f08fd5ae16098e6f326d91ca025bbef removes the whole `gform_id` query parameter bonanza and instead creates a new term meta field for it. If the term meta for it doesn't exist, the button is not shown on the frontend.